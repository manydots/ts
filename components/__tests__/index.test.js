import React, { Component } from 'react';
import { shallow } from 'enzyme';
import { Button } from '../index.tsx';


//expect(Object.keys(Button)).toMatchSnapshot();

describe('Button test', () => {
   const button = shallow(<Button type="warn" message="Button test"  />);
   it('exports  correctly', () => {
     expect(button);
   });
})