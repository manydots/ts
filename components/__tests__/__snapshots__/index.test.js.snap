// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button test exports modules correctly 1`] = `Array []`;
